apps/notifications/src/panel/templates/summary-in-single.jsx

```diff
@@ -16,7 +16,13 @@ class UserHeader extends Component {
 	render() {
 		const grav = this.props.user.media[ 0 ];
 		const grav_tag = <img src={ grav.url } height={ grav.height } width={ grav.width } alt="" />;
-		const home_url = this.props.user.ranges[ 0 ].url;
+		const base_home_url = this.props.user.ranges[ 0 ].url;
+		// Some site notifications populate Id with the siteId, so consider the userId falsy in this
+		// case.
+		const userId =
+			this.props.user.ranges[ 0 ].id !== this.props.user.ranges[ 0 ].site_id &&
+			this.props.user.ranges[ 0 ].id;
```

Comment:
This is for the situation where the site is playing as the user entity for the notification, ex:
![Screenshot 2025-02-18 at 10 32 18 AM](https://github.com/user-attachments/assets/1b4f35b3-1558-4141-8409-30fcb5b8cd2d)
![Screenshot 2025-02-18 at 10 32 24 AM](https://github.com/user-attachments/assets/3930dfb3-da60-4df6-ab3b-e380111ee385)

Otherwise "Addies Garden" would have a userId of the siteId and link to a 404 user profile.

Reply:
Cool! This is a good, simple way to work around this site ID/user ID discrepancy.